.circleci/config.yml

```diff
@@ -95,42 +95,44 @@ jobs:
                 name: Release build llvm
                 command: ./ci/run_travis_commands.sh
 
-    build-starexec-debug:
+    build-fedora-debug:
         docker:
-            - image: usiverify/verify-env:starexec
+            - image: usiverify/verify-env:fedora
               auth:
                 username: mydockerhub-user
                 password: $DOCKERHUB_PASSWORD
               environment:
+                CMAKE_BUILD_TYPE: Debug
+                MODEL_VALIDATION: Dolmen
+                ENABLE_LINE_EDITING: ON
+                PARALLEL: ON
                 FLAGS: -Wall -Wextra -Werror
                 OSMT_INSTALL: ~/osmt-install
-                CMAKE_BUILD_TYPE: Debug
-                MODEL_VALIDATION: pySMT
 
         steps:
             - checkout
             - run:
-                name: Debug build gcc under devtoolset-8
+                name: Debug build gcc in Fedora
                 command: |
-                    cat ./ci/run_travis_commands.sh |scl enable devtoolset-8 bash
+                    eval $(opam env)
+                    ./ci/run_travis_commands.sh
 
-    build-starexec-release:
+    build-fedora-release:
         docker:
-            - image: usiverify/verify-env:starexec
+            - image: usiverify/verify-env:fedora
               auth:
                 username: mydockerhub-user
                 password: $DOCKERHUB_PASSWORD
               environment:
                 CMAKE_BUILD_TYPE: Release
                 FLAGS: -Wall -Wextra -Werror
                 OSMT_INSTALL: ~/osmt-install
-                PARALLEL: ON
+                PARALLEL: OFF
         steps:
             - checkout
             - run:
-                name: Release build gcc under devtoolset-8
-                command: |
-                    cat ./ci/build_maximally_static.sh |scl enable devtoolset-8 bash
+                name: Release build gcc in Fedora
+                command: ./ci/run_travis_commands.sh
```

Comment:
Can we keep using `build_maximally_static.sh` script? It's useful to know we can build fully static executable.

Reply:
Then it should be necessary to install also `static` packages in a docker image. What about making it a separate job that would just build and not run tests?

Reply:
I still don't know what you mean by `static` packages.
It was a job that was just building and not running tests, no?

Reply:
I formerly removed static packages from the Fedora image which were there to allow `build_maximally_static`. I included them again in the image in order to allow this test of static build - as it was before.

Reply:
https://github.com/usi-verification-and-security/verify-env/pull/4/files